modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -211,7 +211,7 @@ public Vector3 Scale
 
         /// <summary>
         /// Returns the <see cref="Basis"/>'s rotation in the form of a
-        /// <see cref="Quaternion"/>. See <see cref="GetEuler"/> if you
+        /// <see cref="Godot.Quaternion"/>. See <see cref="GetEuler"/> if you
```

Comment:
What's the reason for this?

Reply:
Without the namespace, `Quaternion` references the method `Quaternion()`, at least in VSCode.

Reply:
We should investigate if this can be removed in favor of `GetRotationQuaternion` (#51008).

Reply:
- The method `Quaternion()` was added to C# in #11899 (2017), an equivalent did not exist in C++.
- The method `get_rotation_quaternion()` was added to C++ in #19193 (2018).
- The method `GetRotationQuaternion()` was added to C# in #22095 (2018) but as **internal**, it was later made **public** in #42293 (2020).

Reply:
I opened #55675 to fix this confusion.

Reply:
> There are no conversion operators, we could implement explicit ones but this probably applies to all C# types, not just Basis.

I think the conversion from `Basis` to `Quaternion` in GDScript is implicit. I'd prefer we don't add it to C#. I think implicit conversions are bad and should be used only where it improves code (like `NodePath`/`StringName` conversions from string); and the explicit operator adds little value when we already have the constructor.